js/src/wp-seo-admin-global.js

```diff
@@ -293,10 +293,83 @@
 		} );
 	}
 
+	/**
+	 * Make tables scrollable.
+	 *
+	 * Usage: see related stylesheet.
+	 *
+	 * @returns {void}
+	 */
+	function scrollableTables() {
+		$( ".yoast-table-scrollable" ).each( function() {
+			var $table = $( this );
```

Comment:
Just a naming convention so you know it's a jQuery wrapped object. Pretty common in WordPress, have no strong personal preference though.